config/locales/en/layout.yml

```diff
@@ -11,6 +11,7 @@ en:
       sign_out: 'Sign Out'
       admin_panel: 'Administration Panel'
       instance_admin_panel: 'Instance Administration Panel'
+      stop_masquerading: '(Stop Masquerading)'
```

Comment:
Somehow I think removing the parentheses would be more consistent. What do you think?